qiskit/algorithms/factorizers/shor.py

```diff
@@ -57,10 +57,9 @@ class Shor:
     """
 
     @deprecate_function(
-        """The Shor class is deprecated as of Qiskit Terra 0.22.0 and will be removed
-        no sooner than 3 months after the release date.
-        It is replaced by the tutorial at https://qiskit.org/textbook/ch-algorithms/shor.html
-        """,
+        "The Shor class is deprecated as of Qiskit Terra 0.22.0 and will be removed "
+        "no sooner than 3 months after the release date. It is replaced by the tutorial "
+        "at https://qiskit.org/textbook/ch-algorithms/shor.html",
```

Comment:
Why is it necessary to write the deprecation message as a list of strings rather than a multi-line string?

Reply:
This string is also shown as the Python-space warning message, which needs to be a single line.

Reply:
Jake's reason, and also it dramatically simplified the algorithm for modifying the docstring. The issue with this original value is it has a bunch of "                      " embedded in the two followup lines. It was causing failures in the extension from bad indentation.